sources/packages/web/src/components/generic/ModalDialogBase.vue

```diff
@@ -104,9 +104,3 @@ export default {
   },
 };
 </script>
-<style scoped>
-.max-dialog-height {
-  /* Set the max-height to 65% of the viewport to allow scrollable content. */
```

Comment:
When this was introduced, making it based on a percentage of the `vh` it was not the best decision.
To make it more accurate it would be better to have it calculated based on pixes like `min-height: calc(~"100vh - 150px");` where we just need to figure out the correct number to be used instead of `150px`. Can we try it, please? If it works the margin between the modal and top and bottom of the `vh` will be constant.

Reply:
I will address this in my next PR